.taskcluster.yml

```diff
@@ -1,40 +1,47 @@
+#metadata:
+#  name: Taskcluster Generic Worker Tests
+#  description: These tests should ensure that any new commits against generic worker codebase are tested across all supported worker types in gekco.
+#  owner: ${owner} # the user who sent the pr/push e-mail will be inserted here
+#  source: ${event.head.repo.url}  # the repo where the pr came from will be inserted here
 tasks:
-
-
-  ##########################################################
-  ######### Windows Server 2012 R2 Multiuser Build #########
-  ##########################################################
-
-  - provisionerId: pmoore-test
-    workerType: win2012r2-cu
-    metadata:
-      name: "Build/test 64 bit generic-worker (multiuser engine) on Windows Server 2012 R2"
-      description: "This builds and tests the 64 bit Windows version of generic-worker (multiuser engine) on Windows Server 2012 R2"
-      owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
-      source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
-    extra:
-      github:
-        # Events that will trigger this task
-        events:
-          - pull_request.opened
-          - pull_request.synchronize
-          - push
-    scopes:
-      - generic-worker:cache:generic-worker-checkout
-      - secrets:get:repo:github.com/taskcluster/generic-worker
-    payload:
-      features:
-        taskclusterProxy: true
-      maxRunTime: 3600
+  $let:
+    owner: "taskcluster-internal@mozilla.com" # the user who sent the pr/push e-mail will be inserted here
+    repo:
+      $if: 'tasks_for == "github-pull_request" && event["action"] in ["opened", "synchronize"]'
+      then:
+        url: ${event.pull_request.head.repo.clone_url}
+        ref: ${event.pull_request.head.ref}
+        after: ${event.pull_request.head.sha}
+      else:
+        url: ${event.repository.clone_url}
+        ref: ${event.ref}
+        after: ${event.after}
+
+  in:
+    ##########################################################
+    ######### Windows Server 2012 R2 Multiuser Build #########
+    ##########################################################
+    - provisionerId: pmoore-test
+      workerType: win2012r2-cu
+      created: {$fromNow: ""}
+      deadline: {$fromNow: "24 hours"}
+      metadata:
+        name: Build/test 64 bit generic-worker (multiuser engine) on Windows Server 2012 R2
+        description: This builds and tests the 64 bit Windows version of generic-worker (multiuser engine) on Windows Server 2012 R2
+        owner: ${owner} #the user who sent the pr/push e-mail will be inserted here
+        source: ${repo.url}  # the repo where the pr came from will be inserted here
+      scopes:
+        - generic-worker:cache:generic-worker-checkout
+        - secrets:get:repo:github.com/taskcluster/generic-worker
+      payload:
+        features:
+          taskclusterProxy: true
+        maxRunTime: 3600
```

Comment:
It seems to me `command` section (starting on line 45) has the same indentation as `payload` section (starts on line 41)? It should have the same indentation as lines 42 and 44. Same for `artifacts` and `mounts`. All similar spots in the files need to be checked as well

Reply:
indeed, the indentation was completely off! I've updated it, but keep on getting the same error when I try running the validator tool